HW03/EgorChernooky/egor_chernooky_hw_03_t_2.rb

```diff
@@ -0,0 +1,22 @@
+DATE = /\[(.*?)\]/.freeze
+FROM = /^\d{2}\.\d{1}\.\d{3}\.\d{3}/.freeze
+TO = /\/\w*\/\d{1}\/\w*\b/.freeze 
+
+logs = 
+"10.6.246.103 - - [23/Apr/2018:20:30:39 +0300] POST /test/2/messages HTTP/1.1 200 48 0.0498
+10.6.246.101 - - [23/Apr/2018:20:30:42 +0300] POST /test/2/run HTTP/1.1 200 - 0.2277
+2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>
+10.6.246.101 - - [23/Apr/2018:20:31:39 +0300] POST /test/2/messages HTTP/1.1 200 48 0.0290"
+
+def task2(text)
+  output_array = []
+  text.split("\n").select { |line| line.downcase.include?('post')}.map do |line|
+    date = line.scan(DATE).join
+    from = line.scan(FROM).join
+    to = line.scan(TO).join.upcase
+    output_array << "#{date} FROM: #{from} TO: #{to}"
+  end
+  output_array
```

Comment:
Should work without this temp array 
```suggestion
  text.split("\n").select { |line| line.downcase.include?('post')}.map do |line|
    date = line.scan(DATE).join
    from = line.scan(FROM).join
    to = line.scan(TO).join.upcase
    "#{date} FROM: #{from} TO: #{to}"
  end
```

Reply:
> Should work without this temp array

But the task was to return the array so that's why i create it here

Reply:
I'll return an array anyway

Reply:
Done